### ISSUE TITLE
Invalid reference to Temporal::Api::Enums::V1::TaskQueueType::Workflow

### ISSUE THREAD

**Issue by cdimitroulas** [coinbase/temporal-ruby#193]
In the `Temporal::Connection::GRPC#describe_task_queue` method there is a reference to this enum -> `Temporal::Api::Enums::V1::TaskQueueType::Workflow`

A simple bit of code like this seems to suggest that this is an invalid reference as I get an error:
```ruby
connection = Temporal::Connection.generate(Temporal.configuration.for_connection)
puts connection.describe_task_queue(namespace: Temporal.configuration.namespace, task_queue: Temporal.configuration.task_queue)
```

The error:
```
NameError: uninitialized constant Temporal::Api::Enums::V1::TaskQueueType::Workflow
Did you mean?  Temporal::Workflow
               Temporal::Api::Workflow
```

I believe that the correct reference should be `Temporal::Api::Enums::V1::TaskQueueType::TASK_QUEUE_TYPE_WORKFLOW`. When using that instead, then the code above runs as expected. Happy to open a PR if this is indeed the correct fix.

**Comment by antstorm** [comment]
@cdimitroulas yes, I believe you are right and there's a bug there. A fix PR is very welcome!

**Comment by cdimitroulas** [comment]
@antstorm I've opened a PR for it
